### DISCUSSION TITLE
License?

### DISCUSSION THREAD

**Discussion by AlexSegal** [hannesdelbeke/unimenu#49] [General]
Hello @hannesdelbeke! 

I tried unimenu and I really love the neat way you came up with to create menus in different DCCs! Thank you so much! One question though: you did not specify the license (unless I missed it). As far as I understand that prevents me from using the code in the studio environment, since it's not open source by default. It would be great if you could add the license to the project (or point me to one if it exists).

Thanks again!

**Comment by hannesdelbeke** [comment]
done

**Reply by AlexSegal** [reply]
Thanks a ton!
